test-cases/manager/manager-regression-debian9-multiDC-set-distro.yaml

```diff
@@ -0,0 +1,23 @@
+stress_cmd: "cassandra-stress write cl=QUORUM n=1200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"
```

Comment:
I agree with @fgelcer the idea wasn't to duplicate the test yamls, but to create extra yamls for the overrides